### ISSUE TITLE
UI: Tooltip should always be rendered into HTML for accessibility

### ISSUE THREAD

**Issue by rk3rn3r** [debezium/debezium-ui-poc#192]
Currently there's a dynamic tooltip added to the DOM with details on the FAILED connectors. This info is missing for disabled people. We should either provide that additional info to disabled people in a different way for higher accessibility or make the tooltip permanent and only show it dynamically so the content is available to disabled people.

**Comment by indraraj** [comment]
I think this needs to be taken with the PF team, as accessibility is covered within PF.

**Comment by mdrillin** [comment]
Right, accessibility is built into the patternfly components.  In this case, I think the popover component may be better suited for accessibility instead of tooltip.  We could add a clickable infoIcon similar to the popovers on the table selection wizard step, utilizing aria-label.  @nding-anges - is there a preferred approach here?  This is for the connector task/status shown in the screenshot below.

![PopoverVsTooltip](https://user-images.githubusercontent.com/1820403/101812523-15a32c80-3ae1-11eb-9961-c849ff730829.png)

**Comment by nding-anges** [comment]
I believe that accessibility is built into the PatternFly components. Here is the Accessibility guide: https://www.patternfly.org/v4/developer-resources/accessibility-guide/developer-resources/#patternfly-guidelines
Otherwise, I think the pop-over will be helpful at this point.

**Comment by mdrillin** [comment]
Pending PR #234 - removed the tooltip altogether on the ConnectorStatus, since the tooltip there didn't provide any additional info.  On the TaskStatus, changed the tooltip to a popover.  The popover will only show up if there are additional error messages available.  The popover should better address the accessibility.  There will be additional work on the TaskStatus component in the future, as we add more capabilities.

**Comment by indraraj** [comment]
pr merged.
